### PR TITLE
Feature/accordion

### PR DESCRIPTION
**Pull request by myktybek03**

### CODE REVIEW

File: src/layout/user/FrequentlyAskedQuestions.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            className="accordion"
+            {...item}
+          />
+        ))}
+      </AccordionBoxStyled>
+    </ContainerStyled>
+  )
+}
+
+export default FrequentlyAskedQuestions
```
Comment (FeruzaKazakova) [inline src/layout/user/FrequentlyAskedQuestions.tsx:100]: кажется есть конфликты try to solve the problem

File: src/layout/user/FrequentlyAskedQuestions.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            expanded={expanded}
+            onChange={() => handleChange(item.background)}
+            className="accordion"
+            {...item}
+          />
+        ))}
+      </AccordionBoxStyled>
+    </ContainerStyled>
+  )
+}
```
Comment (FeruzaKazakova) [inline src/layout/user/FrequentlyAskedQuestions.tsx:98]: no comments

**Review by FeruzaKazakova — APPROVED** [review]

**Review by Nurzada03 — APPROVED** [review]

**Review by dilmuratsarybaev — APPROVED** [review]

File: src/assets/icons/CheckIcon.svg
Comment (songhee24) [inline src/assets/icons/CheckIcon.svg:1]: icon is not component please change the name to checkIcon

Comment (songhee24) [reply src/assets/icons/CheckIcon.svg:1]: also change it everywhere

**Review by songhee24 — CHANGES_REQUESTED** [review]

**Review by songhee24 — APPROVED** [review]

**Pull request merged**